packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -199,18 +199,51 @@ mixin SchedulerBinding on BindingBase, ServicesBinding {
 
     if (!kReleaseMode) {
       int frameNumber = 0;
-
-      // use frameTimings. https://github.com/flutter/flutter/issues/38838
-      // ignore: deprecated_member_use
-      window.onReportTimings = (List<FrameTiming> timings) {
+      addTimingsCallback((List<FrameTiming> timings) {
         for (FrameTiming frameTiming in timings) {
           frameNumber += 1;
           _profileFramePostEvent(frameNumber, frameTiming);
         }
+      });
+    }
+  }
+
+  /// Add a [TimingsCallback] that receives [FrameTiming] sent from the engine.
+  ///
+  /// This can be used, for example, to monitor the performance in release mode,
+  /// or to get a signal when the first frame is rasterized.
```

Comment:
Should this maybe also say that using this is preferred over window.onReportTimings because it allows for multiple callbacks?